checkov/sast/record.py

```diff
@@ -0,0 +1,88 @@
+from typing import List, Dict, Any, Tuple, Optional
+
+from termcolor import colored
+
+from checkov.common.bridgecrew.severities import Severity
+from checkov.common.models.enums import CheckResult
+from checkov.common.output.record import Record
+from checkov.common.typing import _CheckResult
+
+'''
+        check_class: str,
```

Comment:
what is the meaning of this documentation?
Also if you add doc to the entire module it should be before the imports in my opinion

Reply:
That's how the original record file is written but I'm not sure what's the my meaning and it really doesnt look necessary, deleting it